From: Daniel Drake <dsd@gentoo.org>
Date: Mon, 19 Jan 2009 14:36:41 +0000 (+0100)
Subject: libxt_owner: add more spaces to output
X-Git-Url: https://git.netfilter.org/cgi-bin/gitweb.cgi?p=iptables.git;a=commitdiff_plain;h=cfb9cf0509ad8100cd8d6ab52d60a8ffbb318578

libxt_owner: add more spaces to output

Commit bb9284d1 ("libxt_owner: add spaces to output") moved the printing
of spaces away from the owner_mt_print() function family, but forgot to
add spaces in all of the print_item() functions that are called.

This is likely to be at least the partial cause of
https://bugs.gentoo.org/show_bug.cgi?id=254435

Signed-off-by: Daniel Drake <dsd@gentoo.org>
Approves-of-this-patch: Jan Engelhardt <jengelh@medozas.de>
Signed-off-by: Patrick McHardy <kaber@trash.net>
---

diff --git a/extensions/libxt_owner.c b/extensions/libxt_owner.c
index 298075e..4cd173e 100644
--- a/extensions/libxt_owner.c
+++ b/extensions/libxt_owner.c
@@ -324,7 +324,7 @@ owner_mt_print_item_v0(const struct ipt_owner_info *info, const char *label,
 		return;
 	if (info->invert & flag)
 		printf("! ");
-	printf(label);
+	printf("%s ", label);
 
 	switch (info->match & flag) {
 	case IPT_OWNER_UID:
@@ -375,7 +375,7 @@ owner_mt6_print_item_v0(const struct ip6t_owner_info *info, const char *label,
 		return;
 	if (info->invert & flag)
 		printf("! ");
-	printf(label);
+	printf("%s ", label);
 
 	switch (info->match & flag) {
 	case IP6T_OWNER_UID:
